nh_eobs_mental_health/models/nh_clinical_sql.py

```diff
@@ -367,29 +367,44 @@ def get_refused_wardboard(self, interval):
 
     def get_collect_activities_sql(self, activity_ids_sql):
         sql = self.collect_activities_skeleton.replace(
-            'left join ews1 on ews1.spell_activity_id = spell.id',
-            'left join ews1 on ews1.spell_activity_id = spell.id '
+            'left join ews1 on ews1.spell_activity_id = spell_activity.id',
+            'left join ews1 on ews1.spell_activity_id = spell_activity.id '
+            'LEFT JOIN nh_clinical_spell AS spell '
```

Comment:
Joining with the spell and PME tables so I can then conditionally join with the refused table (using the spell.move_date to filter out transfers and the date_terminated of the last PME for obs_stop/restart)